config.py

```diff
@@ -1,198 +1,42 @@
-"""
```

Comment:
This config file and `knack_run_jvd_report` were unfinished work from back when we though we'd try to auto-generate an Excel file for Finance folks. it never materialized.